art/estimators/classification/tensorflow.py

```diff
@@ -1027,6 +1078,33 @@ def get_activations(self, x, layer, batch_size=128):
         else:
             return None
 
+    def custom_gradient(self, nn_function, tensors, input_values):
+        """
+        Returns the gradient of the nn_function with respect to model input
+
+        :param nn_function: an intermediate tensor representation of the function to differentiate
+        :type nn_function: a Keras tensor
+        :param tensors: the tensors or variables to differentiate with respect to
+        :type tensors: `list`
+        :param input_values: the inputs to evaluate the gradient
+        :type input_values: `list`
+        :return: the gradient of the function w.r.t vars
+        :rtype: `np.ndarray`
+        """
+        raise NotImplementedError
+        # import tensorflow as tf
+        # with tf.GradientTape() as g:
+        #     g.watch(tensors[0])
+        #
+        # grads = g.gradient(nn_function, tensors[0])
+        # return grads.numpy()
+        # # outputs = tf.function(tensors, [grads])
+        # # return outputs(input_values)
+
+    def get_input_layer(self):
+        raise NotImplementedError
+        # return self._model.layers[0].input
+
```

Comment:
Please remove these lines because custom_gradient is so far only part of KerasClassifier and not the Estimator API.
```suggestion
```